student/src/test/java/tds/student/services/remote/RemoteItemScoringServiceTest.java

```diff
@@ -0,0 +1,142 @@
+package tds.student.services.remote;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import tds.itemrenderer.data.IITSDocument;
+import tds.itemscoringengine.ItemScore;
+import tds.student.services.abstractions.IItemScoringService;
+import tds.student.sql.data.IItemResponseScorable;
+import tds.student.sql.data.ItemResponseScorable;
+import tds.student.sql.data.ItemResponseUpdate;
+import tds.student.sql.data.OpportunityInstance;
+import tds.student.sql.repository.remote.ItemScoringRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RemoteItemScoringServiceTest {
+
+    @Mock
+    private IItemScoringService mockLegacyItemScoringService;
+
+    @Mock
+    private ItemScoringRepository mockRepository;
+
+    private RemoteItemScoringService service;
+
+    @Before
+    public void setup() {
+        service = new RemoteItemScoringService(mockLegacyItemScoringService, true, true, mockRepository);
+    }
+
+    @Test
+    public void itShouldUseRepositoryAndLegacyToUpdateResponsesWhenEnabled() throws Exception {
+        final OpportunityInstance oppInstance = oppInstance();
+        final List<ItemResponseUpdate> responsesUpdated = newArrayList(mock(ItemResponseUpdate.class));
+        final Float pageDuration = 1.23f;
+
+        service.updateResponses(oppInstance, responsesUpdated, pageDuration);
+
+        verify(mockLegacyItemScoringService).updateResponses(eq(oppInstance), eq(responsesUpdated), eq(pageDuration));
+        verify(mockRepository).updateResponses(
+            eq(oppInstance.getExamId()),
```

Comment:
good point, since they're all equality matchers, I could just use the values.  Habit.